src/lib/swapper/swappers/ThorchainSwapper/utils/getSignTxFromQuote.ts

```diff
@@ -78,6 +79,16 @@ export const getSignTxFromQuote = async ({
     }
 
     case CHAIN_NAMESPACE.CosmosSdk: {
+      const maybeThornodeQuote = await getQuote({
```

Comment:
This call feels a bit odd. `getSignTxFromQuote` receives a `TradeQuote` object which, in the case of THORSwap, is already derived from the response from the `getQuote()` 
 (`src/lib/swapper/swappers/ThorchainSwapper/utils/getQuote/getQuote.ts)` function we are calling here.

It seems like a duplicate network request that we should be able to avoid.

If the purpose is for catching errors returned from this function, we should probably handle them in `src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts` and bubble them there. 

It does, however, seem like the response data is now being used further down the tree (e.g. in `getLimit` - I wonder if there is another way to get that data there without duplicating requests.

Reply:
You're right, it's a duplicate network request that has been lifted out of `getLimit` :) But I lifted it out of there because `getTradeQuote` was doing a duplicate request also (it calls `getLimit` too lol), so I modified it to pass it in. 

So now we only have 1 duplicate request, and preferably I'd keep cleaning this up out of scope if that's cool?

Reply:
Yeah sure, it's no worse than it was :lolsob: